cpplib/src/Functions/PythonInteface.cpp

```
@@ -143,6 +144,55 @@ PyObject* create_list_from_points(const std::vector<cpplib::geometry::Point<IT>>
 	return o_ret;
 }
 
+template <std::integral T>
+PyObject* create_list_from_vector_int(const std::vector<T>& vec) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)